docs/organizations/custom-organization-switcher.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Create a custom organization switcher
+description: Learn how to build a custom organization switcher using the useOrganizationList() hook.
+---
+
+# Create a custom organization switcher
+
+If you need more control than what Clerk's prebuilt [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) component provides, you can build your own organization switcher using the [`useOrganizationList()`](/docs/references/react/use-organization-list) hook.
+
+The example below is written for Next.js App Router, but it is supported by any React meta framework, such as Remix or Gatsby. 
+
+To get started, create a new file called `OrganizationSwitcher.tsx` and paste the following code into it. You can place this file anywhere in your project, but in this example, it is placed in a folder called `components`.
+
+```tsx filename="app/components/OrganizationSwitcher.tsx"
+"use client"
+
+import { useOrganizationList } from "@clerk/nextjs";
+
+export const CustomOrganizationSwitcher = () => {
+  const { isLoaded, setActive, userMemberships } = useOrganizationList({
+    userMemberships: {
+      infinite: true,
+    },
+  });
+
+  if (!isLoaded) {
+    return <>Loading</>;
```

Comment:
This seems slightly weird, maybe a `<p>`?

Reply:
good catch!